server/container/guice/cassandra-guice/src/test/resources/listeners.xml

```diff
@@ -19,9 +19,7 @@
  -->
 
 <listeners>
-  <listener>
-    <class>org.apache.james.mailbox.spamassassin.SpamAssassinListener</class>
```

Comment:
why?

Reply:
As no spamassassin is running during the tests, it avoid having a lot of stacktrace about timeout while trying to connect to it.

Reply:
I guess it depends on which test you run, right? There should be integration tests that require that

Reply:
there seems to be none at the time being.

Reply:
JMAP-draft tests, inMemory